packages/cysync-core/src/dialogs/Send/context/index.tsx

```diff
@@ -228,7 +236,7 @@ export const SendDialogProvider: FC<SendDialogContextProviderProps> = ({
   );
 
   useEffect(() => {
-    if (disableAccountSelection) goTo(1, 0);
+    if (disableAccountSelection || skipAccountSelection) goTo(1, 0);
```

Comment:
Does `skipAccountSelection` works exactly the same as `disableAccountSelection`? If so, why don't we just use `disableAccountSelection` instead?

Reply:
disableAccountSelection carries forward to other tabs like recipient where it is used to disable inputs as well like address, amount, fee. But here we dont want that